Add Fenced Frames OT information (#2807)
GoogleChrome/developer.chrome.com: site/en/docs/privacy-sandbox/fenced-frame/index.md

authors:
  - jackjey
  - alexandrawhite
  - kevinkiklee
date: 2022-03-07
updated: 2022-05-10
---

## Implementation status

This document outlines a proposal for a new HTML element: `<fencedframe>`.

> * Fenced frames are available for experimentation in the [Privacy Sandbox unified origin trial](/origintrials/#/view_trial/771241436187197441) from [M102 to M105](https://chromiumdash.appspot.com/schedule). Learn how you can [setup your origin trial](/blog/privacy-sandbox-unified-origin-trial/) and [join us for a feedback/discussion](https://github.com/WICG/fenced-frame/issues). 

Reviewer: ```suggestion
*  Experiment with fenced frames in the [Privacy Sandbox unified origin trial](/origintrials/#/view_trial/771241436187197441) from M102 to M105. Learn how to [set up the origin trial](/blog/privacy-sandbox-unified-origin-trial/) and [join us for a feedback/discussion](https://github.com/WICG/fenced-frame/issues).
```
